From: Michael Davidsaver <mdavidsaver@bnl.gov>
Date: Wed, 19 Sep 2012 10:16:26 -0400
Subject: [PATCH] allow compiler flags from commandline

make USR_CFLAGS=xxxx

may override flags forced by some modules

make CMD_CFLAGS=xxxx

should add to them if modules correctly add to USR_CFLAGS.

eg.

USR_CFLAGS += yyyy
---
 configure/CONFIG_COMMON |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure/CONFIG_COMMON b/configure/CONFIG_COMMON
index a26f077..297ec46 100644
--- a/configure/CONFIG_COMMON
+++ b/configure/CONFIG_COMMON
@@ -198,11 +198,11 @@ OP_SYS_INCLUDES =
 
 # Makefile specific flags
 USR_INCLUDES =
-USR_CFLAGS =
-USR_CXXFLAGS =
+USR_CFLAGS = $(CMD_CFLAGS)
+USR_CXXFLAGS = $(CMD_CXXFLAGS)
 USR_LDFLAGS =
 USR_LIBS =
-USR_CPPFLAGS =
+USR_CPPFLAGS = $(CMD_CPPFLAGS)
 USR_DBDFLAGS =
 USR_ARFLAGS =
 
-- 
